src/metrics/prometheus.go

```diff
@@ -24,6 +26,23 @@ func Push(config *core.Configuration) {
 		return
 	}
 
+	if config.Metrics.PushHostinfo {
+		name, _ := os.Hostname()
+		gauge := prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
why a gauge rather than a counter?

Reply:
Re-considered this, counter probably does make more sense from the perspective of gathering and summing them.